core/src/driver/stablex_driver.rs

```diff
@@ -586,4 +645,68 @@ mod tests {
             .unwrap()
             .is_ok());
     }
+
+    #[test]
+    fn burnt_fees_ok() {
```

Comment:
nit: Since there are multiple ways how burnt fees could be computed (we could also price all buy volumes in OWL and take 0.1% of it) I think it would be nice if the test could document in its name what ok means.

E.g. `burnt_fees_is_half_imbalance_of_reference_token_sold_and_bought`